example/ranking/web-service.ts

```diff
@@ -0,0 +1,38 @@
+import { env, WebServiceBuilder, WebService, EnvVar } from "./deps.ts";
+
+const replicas = {
+  staging: 1,
+  production: 2,
+};
+
+const container = "ranking";
+
+const envs: Array<EnvVar> = [
+  {
+    name: "RS_ROLE",
+    value: "api",
+  }
+]
+
+const args: Array<string> = [
+  "-J-XX:InitialRAMPercentage=25.0",
+  "-J-XX:MaxRAMPercentage=80.0",
+  "-J-XX:+PreserveFramePointer",
+  "-Drs.api.ports.grpc=8000",
+]
+
+const res: WebService = WebServiceBuilder
+  .create(container, "370025973162.dkr.ecr.us-east-2.amazonaws.com/ranking:2.13.5", replicas[env])
```

Comment:
Ranking contains much info related to Tubi's internal stuff. I think it's better to create another repo under [tubi](https://github.com/adRise) for that.